src/TestClients/AnalyticsTestClient/Menus/ResultsApiConsumerMenu.cs

```diff
@@ -22,34 +22,13 @@ public ResultsTimeSpanMenu()
             {
                 Title = "Latest Results (FS)";
 
-                var clusteringSerializer = new CsvMessageFormatter("id", "type", "version", "enqueueTimeUtc", "gameSession", "lat", "lon", "geoHash", "geoHashPrecision", "geoHashCenterLat", "geoHashCenterLon", "rnd");
-                var dauSerializer = new CsvMessageFormatter("id", "type", "version", "enqueueTimeUtc", "gameSession", "gamerTag");
-
-
-                MenuItems.Add('1', new ConsoleMenuItem("DAU", () => GetResults(dauSerializer, "dau", "session-start")));
-                MenuItems.Add('2', new ConsoleMenuItem("Clustering", () => GetResults(clusteringSerializer, "clustering", "geo-location")));
+                // TODO: WIP
+                //MenuItems.Add('1', new ConsoleMenuItem("DAU", () => GetResults(dauSerializer, "dau", "session-start")));
+                //MenuItems.Add('2', new ConsoleMenuItem("Clustering", () => GetResults(clusteringSerializer, "clustering", "geo-location")));
             }
 
             private void GetResults(IMessageFormatter formatter, string pipeline, string messageType)
             {
-                var start = GetDate("Start date");
-                var end = GetDate("End date");
-
-                if (start == null || end == null) return;
-
-                var filePathAlgorithm = new DateFolderStructure(newFileOption: NewFileNameOptions.Every5Minutes);
-
-                IFolderStructure _filePathAlgorithm = new DateFolderStructure(newFileOption: NewFileNameOptions.Every5Minutes);
-
-                Console.Write("Root directory: ");
-
-                var path = Console.ReadLine();
-
-                var f = new Nether.Analytics.FileResultsReader(formatter, _filePathAlgorithm, path, pipeline, messageType);
-                foreach (var msg in f.Get(start.Value, end.Value))
-                {
-                    Console.WriteLine(msg.ToString());
-                }
             }
 
             private DateTime? GetDate(string type)
```

Comment:
There is a class and method called ConsoleEx.ReadLine(string, DateTime), that would fit you well here that we are using otherwise. Please have a look at that one and improve that one if needed to.